### PR TITLE
Bugfix/warnings suppression

### PR DESCRIPTION
**Pull request by Zoolixx**

### CODE REVIEW

**Comment by khenderick** [comment]
Please rebase your changes on development. It's hard to review when mixed with changes from another PR/branch

File: src/containers/led-global.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ export class GlobalLed extends BaseObject {
         };
     }
 
+    @computedFrom('i18n', 'id')
```
Comment (khenderick) [inline src/containers/led-global.js]: I think you can get rid of the `i18n` here. Feel free to give it a test and check whether changing the language updates the test.

File: src/containers/sensor.js [hunk 2]
```diff
@@ -42,8 +42,10 @@ export class Sensor extends BaseObject {
         };
     }
 
+    @computedFrom('name')
     get inUse() {
-        return this.name !== '' && this.name !== 'NOT_IN_USE'
+        var result = (this.name !== '') && (this.name !== 'NOT_IN_USE')
```
Comment (khenderick) [inline src/containers/sensor.js]: Please return immediately, no need to store it in a variable.

File: src/containers/shutter.js [hunk 3]
```diff
@@ -64,15 +65,15 @@ export class Shutter extends BaseObject {
         }
         return this.inUse ? this.name : this.id.toString();
     }
-
+    @computedFrom('upDownConfig')
```
Comment (khenderick) [inline src/containers/shutter.js]: Also include `id` here.

File: src/containers/thermostat.js [hunk 4]
```diff
@@ -111,19 +111,21 @@ export class Thermostat extends BaseObject {
         };
     }
 
+    @computedFrom('sensorId')
     get isRelay() {
-        return this.sensorId === 240;
+        var result = this.sensorId === 240;
```
Comment (khenderick) [inline src/containers/thermostat.js]: Immediately return.

File: src/pages/settings/users.js [hunk 5]
```diff
@@ -142,8 +142,7 @@ export class Users extends Base {
     }
 
     @computedFrom(
-        'activeUser', 'activeUser.acl', 'activeUser.acl.edit', 'aciveUser.acl.edit.allowed',
-        'activeUser.role', 'activeUser.role.acl', 'activeUser.role.acl.edit', 'aciveUser.role.acl.edit.allowed',
+        'activeUser', 'activeUser.role', 'aciveUser.acl.edit.allowed', 'aciveUser.role.acl.edit.allowed',
```
Comment (khenderick) [inline src/pages/settings/users.js]: Are you sure that you don't need to specify the intermediate objects? You e.g. specify `activeUser.acl.edit.allowed`, but don't you have to also include `activeUser.acl` and `activeUser.acl.edit`?

Comment (khenderick) [reply src/pages/settings/users.js]: If you need them; please re-add them (and also at other changes).
If you don't need them; please take a look at all the other `@computedFrom`s to remove them there as well.

Comment (Zoolixx) [reply src/pages/settings/users.js]: Turns out we don't need to specify all objects' properties.
Changes have been made here: 0edcb81918c3426377598d7ee68665df2c7d4647

File: src/containers/thermostat.js (hunk 4 above)
Comment (Zoolixx) [reply src/containers/thermostat.js]: Changes have been made here: 0edcb81918c3426377598d7ee68665df2c7d4647

File: src/containers/shutter.js (hunk 3 above)
Comment (Zoolixx) [reply src/containers/shutter.js]: Changes have been made here: 0edcb81918c3426377598d7ee68665df2c7d4647

File: src/containers/sensor.js (hunk 2 above)
Comment (Zoolixx) [reply src/containers/sensor.js]: Changes have been made here 0edcb81918c3426377598d7ee68665df2c7d4647

File: src/containers/led-global.js (hunk 1 above)
Comment (Zoolixx) [reply src/containers/led-global.js]: Changes have been made here: 0edcb81918c3426377598d7ee68665df2c7d4647

**Pull request merged**